featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -184,6 +184,42 @@ def label_func(df):
             feature_matrix['label_func']).values.all()
 
 
+def test_cfm_compose_approximate(es):
+    if not all(isinstance(entity.df, pd.DataFrame) for entity in es.entities):
+        pytest.xfail('dask does not support approximate')
+
+    def label_func(df):
+        return df['value'].sum() > 10
+
+    lm = cp.LabelMaker(
+        target_entity='id',
+        time_index='datetime',
+        labeling_function=label_func,
+        window_size='1m'
+    )
+
+    df = es['log'].df
+    df = to_pandas(df)
+    labels = lm.search(
+        df,
+        num_examples_per_instance=-1
+    )
+    labels = labels.rename(columns={'cutoff_time': 'time'})
```

Comment:
These exact label times are used in separate test as well.  Perhaps we should convert them to a fixture?